src/components/ContributionsEpic.js.ts

```diff
@@ -0,0 +1,90 @@
+// The function returned here is going to be called on the platform and run
```

Comment:
This code isn't transpilled to ES5 so we need to make sure we don't use syntax features not supported by older browsers (e.g. arrow functions). Constants have partial support on IE11.

Reply:
None of my business but that sounds tricky to maintain and enforce! If it's intended to run on the client as well as Node, why not transpile it so it's guaranteed to run in both environments?

Reply:
Hey @SiAdcock - what I meant by that is the code is transpiled to ES9, not ES5, because this is server-side rendered and the version of Node we use has support for the 2018 features. But now we have this one-off use case where a little snippet of our code is meant to run on the client and so ES2018 is compatible enough. What do we do?

1) Transpile everything down to ES5 and accept a slight performance hit across all platforms? or

2) Avoid the need to change transpilation settings by making the snippet slightly more IE11 friendly (I realised I could do this by just using function statements instead of arrow functions).

Initially I chose 2 because this is a short-term solution and not something we'll ever expand on or adopt as a standard pattern. We're not going to ask other developers to write or serve code this way.

However, after sleeping on this, I changed my mind and decided to transpile everything down to ES5, because:
- It's true that this is a tricky pattern to maintain/enforce
- We don't know how "short" the short-term is
- We don't serve any other JS from this service yet so the bandwidth increase from ES2018 code to ES5 is small and self-contained.
- Developers shouldn't care about specific feature support across browsers when writing JS.

So basically, I've tweaked the tsconfig settings and the JS snippet to rely on ES5 transpilation until we move away from this approach.